spyrmsd/graph.py

```diff
@@ -83,14 +81,14 @@ def adjacency_matrix_from_atomic_coordinates(
     A = np.zeros((n, n))
 
     for i in range(n):
-        r_i = qcel.covalentradii.get(atomicnums[i], units="angstrom")
+        r_i = constants.anum_to_covalentradius[atomicnums[i]]
 
         for j in range(i + 1, n):
-            r_j = qcel.covalentradii.get(atomicnums[i], units="angstrom")
+            r_j = constants.anum_to_covalentradius[atomicnums[j]]
```

Comment:
The wrong atomic number (and therefore covalent radius) was selected here. This is now fixed and a test spotting this problem has been added.